code/datums/brain_damage/mild.dm

```diff
@@ -0,0 +1,278 @@
+	var/list/trigger_turfs = list()
+	var/list/trigger_species = list()
+
+/datum/brain_trauma/mild/phobia/New(mob/living/carbon/C, _permanent, specific_type)
+	phobia_type = specific_type
+	if(!phobia_type)
+		phobia_type = pick("spiders", "space", "security", "clowns", "greytide", "lizards")
+
+	gain_text = "<span class='warning'>You start finding [phobia_type] very unnerving...</span>"
+	lose_text = "<span class='notice'>You no longer feel afraid of [phobia_type].</span>"
+	scan_desc += " of [phobia_type]"
+	switch(phobia_type)
+		if("spiders")
+			trigger_words = list("spider","web","arachnid")
+			trigger_mobs = list(/mob/living/simple_animal/hostile/poison/giant_spider)
+			trigger_objs = list(/obj/structure/spider) //includes webs and spiderlings
+		if("space")
+			trigger_words = list("space", "star", "universe", "void")
+			trigger_turfs = list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)
+		if("security")
+			trigger_words = list(" sec ", "security", "shitcurity", "stunbaton", "taser", "beepsky")
+			trigger_mobs = list(/mob/living/simple_animal/bot/secbot)
+			trigger_objs = list(/obj/item/clothing/under/rank/security, /obj/item/clothing/under/rank/warden, /obj/item/clothing/under/rank/head_of_security,\
+				/obj/item/clothing/under/rank/det, /obj/item/melee/baton, /obj/item/gun/energy/taser, /obj/item/restraints/handcuffs, /obj/machinery/door/airlock/security)
+		if("clowns")
+			trigger_words = list("clown", "honk", "banana", "slip")
+			trigger_objs = list(/obj/item/clothing/under/rank/clown, /obj/item/clothing/shoes/clown_shoes, /obj/item/clothing/mask/gas/clown_hat,\
+				/obj/item/device/instrument/bikehorn, /obj/item/device/pda/clown, /obj/item/grown/bananapeel)
+		if("greytide")
+			trigger_words = list("assistant", "grey", "gasmask", "gas mask", "stunprod", "spear", "revolution", "viva")
+			trigger_objs = list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/cattleprod, /obj/item/twohanded/spear,\
+				/obj/item/clothing/mask/gas)
+		if("lizards")
+			trigger_words = list("lizard", "ligger", "hiss", " wag ")
+			trigger_objs = list(/obj/item/toy/plush/lizardplushie, /obj/item/reagent_containers/food/snacks/kebab/tail, /obj/item/organ/tail/lizard,\
+				/obj/item/reagent_containers/food/drinks/bottle/lizardwine)
+			trigger_mobs = list(/mob/living/simple_animal/hostile/lizard) //they're hostile! of course they're scary!
+			trigger_species = list(/datum/species/lizard)
+		if("spooky skeletons")
+			trigger_words = list("skeleton", "milk", "xylophone", "bone", "calcium", "the ride never ends")
+			trigger_objs = list() // todo: find paths for all above
+			trigger_mobs = list() // i forget if we have
+			trigger_species = list(/datum/species/skeleton)
+
+	trigger_turfs = typecacheof(trigger_turfs)
+	trigger_mobs = typecacheof(trigger_mobs)
+	trigger_objs = typecacheof(trigger_objs)
+	trigger_species = typecacheof(trigger_species)
```

Comment:
These typecaches should be static somewhere. Either static in here or if they're used elsewhere a global list

Reply:
unless i make a separate global list for every trigger type for every phobia i can't do that, i think, and i'd rather not clutter global vars needlessly

Reply:
It's not needless, it's better than the performance cost here and you could just put it in a subsystem

Reply:
Or for that matter a list of lists

Reply:
@XDTM you still need to deal with this